### PR TITLE
Allow setting membership type in influx org members add

### PR DESCRIPTION
**Pull request by candrewlee14**
This allows listing, adding, and removing owners from an org with `influx org members <command>`.
- Listing will list owners and members.
- Adding will add member by default, or owner with `--owner` flag.
- Removing will remove owners and members, no flag needed.

Closing #231

### CODE REVIEW

File: cmd/influx/org_members.go [hunk 1]
```diff
@@ -45,6 +45,11 @@ func newOrgMembersAddCmd() cli.Command {
 				EnvVar:      "INFLUX_ORG_ID",
 				Destination: &params.OrgID,
 			},
+			&cli.BoolFlag{
+				Name:        "owner",
```
Comment (DStrand1) [inline cmd/influx/org_members.go:49]: maybe allow `o` here as well for convenience?

**Comment by samhld** [comment]
Seems reasonable to me! If we want to fully satisfy the original issue, we could add a separate hint to point the user at the UI. I imagine most users would know about it but I could see them not. The question would be, can we point them at the exact address of where the UI is? Any discovery tooling to point them exactly where to go to see the UI?

**Review by DStrand1 — APPROVED** [review]
LGTM! Just one small suggestion for the CLI command usage

File: cmd/influx/org_members.go (hunk 1 above)
Comment (candrewlee14) [reply cmd/influx/org_members.go:49]: I wondered if users might accidentally use `-o` thinking they were passing an org flag, but actually set the user as owner. What do you think @samhld ?

**Pull request merged**